tasks/install_tasks.py

```diff
@@ -102,14 +131,14 @@ def install_protoc(ctx, version="26.1"):
     artifact_url = f"https://github.com/protocolbuffers/protobuf/releases/download/v{version}/protoc-{version}-{platform_os}-{platform_arch}.zip"
     zip_path = "/tmp"
     zip_name = "protoc"
-    zip_file = ospath.join(zip_path, f"{zip_name}.zip")
+    zip_file = os.path.join(zip_path, f"{zip_name}.zip")
 
     gh = GithubAPI(public_repo=True)
     # the download_from_url expect to have the path and the name of the file separated and without the extension
     gh.download_from_url(artifact_url, zip_path, zip_name)
 
     # Unzip it in the target destination
-    destination = ospath.join(Path.home(), ".local")
+    destination = os.path.join(Path.home(), ".local")
```

Comment:
NIT
```suggestion
    destination = Path.home() / ".local"
```

Reply:
I find the suggested version a bit more cryptic 😄